src/AppInstallerCLICore/Command.cpp

```diff
@@ -43,6 +44,21 @@ namespace AppInstaller::CLI
         {
             m_fullName = name;
         }
+
+        if (!aliases.empty())
+        {
+            for (auto& alias : aliases)
+            {
+                std::string _aliasFullName;

```

Comment:
Nit: I think the leading underscore is not consistent with the style in the rest of the project